vpr/src/base/constraints_load.cpp

```diff
@@ -0,0 +1,106 @@
+#include "constraints_load.h"
+
+void print_region(FILE* fp, Region region);
```

Comment:
could include as helper functions in VprConstraints class since they are tightly related to the class

Reply:
could also have declarations in the class and keep implementations here

Reply:
when moving to class, make sure to include comments (used for echo files, etc)

Reply:
if keeping the functions in this file, keep declaractions here too, with static keyword